CHANGELOG.md

```diff
@@ -12,6 +12,14 @@ official [coreboot repository](https://review.coreboot.org/cgit/coreboot.git)
 Please use [pce-fw-builder](https://github.com/pcengines/pce-fw-builder)
 
 ## [Unreleased]
+
+## [v4.16.0.1] - 2022-02-11
+### Changed
+- Rebased with official coreboot repository commit b4ba289f
```

Comment:
we changed the markers around the debug prints in coreboot, this also qualifies as a change

Reply:
I have added an information about loglevel prefixes and ANSI escapes being disabled.